recipes/cpptrace/all/conanfile.py

```diff
@@ -43,7 +43,10 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("libdwarf/0.8.0")
+        if Version(self.version) >= Version("0.4.0"):
+            self.requires("libdwarf/[>=0.9.1 <20191104]")
+        else:
+            self.requires("libdwarf/[>=0.8.0 <0.9.0 <20191104]")
```

Comment:
```suggestion
        if Version(self.version) >= Version("0.4.0"):
            self.requires("libdwarf/[>=0.9.1 <20191104]")
        else:
            self.requires("libdwarf/[>=0.8.0 <0.9.0 <20191104]")
```

Please, do not use version range for libdwarf. Only few versions are allowed to use it in CCI:

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges

There is the risk changing the recipe revision, resulting in missing packages. In small steps we are allowing more packages to use it.

Reply:
Thanks!